examples/simple_daplink/simple_daplink.ino

```diff
@@ -311,16 +120,14 @@ void setup() {
     USB_ResponseIdle = 1;
     free_count = FREE_COUNT_INIT;
     send_count = SEND_COUNT_INIT;
-
-    USBDevice.getSerialDescriptor(Serial_u16);
-
 }
 
 
 void loop() {
   // put your main code here, to run repeatedly:
-  baud = Serial.getBaud();
+  baud = Serial.baud();
   if (baud != old_baud) {
+    SerialTTL.end();
```

Comment:
I think we need to end() hardware uart first, before re-begin().

Reply:
`end()` Is it a new API that was recently added? We didn't know that.

Reply:
https://www.arduino.cc/reference/en/language/functions/communication/serial/end/ it is optional, maybe we don't need it as all. I could revert this change.